package.json

```diff
@@ -109,6 +109,7 @@
     "helmet": "3.8.1",
     "hpp": "0.2.2",
     "html-webpack-plugin": "2.30.1",
+    "http-proxy-middleware": "0.17.4",
```

Comment:
@osk I removed this rule `'import/no-extraneous-dependencies': 0`, from our default config, not quite sure if it was really useful, does it sound good to you?

Reply:
Yes, sounds good.